globus_sdk/config/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .env_vars import get_environment_name, get_http_timeout, get_ssl_verify
```

Comment:
Is there a particular reason for the relative imports?

Reply:
We actually have it in our style guide to always use absolute imports, but that note is a little dated. We were coming from an era (reaching back to py2.4, 2.5, 2.6) in which relative import behaviors weren't very consistent / good.

Now that we're in pure python3, I was considering inverting that stance. Relative imports make refactoring easier, since you can rename an entire subpackage and all of its components move gracefully. That's the main advantage, and I'm not aware of any particular advantages of absolute imports in the python3 world.